scripts/eve-parity.py

```diff
@@ -60,6 +60,8 @@ def unmapped_fields(keywords, keys):
     for key in keys.keys():
         if "keywords" not in keys[key]:
             with_missing.add(key)
+        elif keys[key]["keywords"] is False:
+            with_missing.add(key)
```

Comment:
Should this be an empty list rather than false?

`dns.version` is `false`  https://github.com/OISF/suricata/blob/master/etc/schema.json#L1261 as its determined to be not applicable to a keyword. This change makes it show up in `unmapped-fields`, but it shouldn't. This is our sign that its dealt with, no keyword needed. Therefore it should not show up in `unmapped-fields`, or `mapped-fields`.